eth_worker/strip_contracts_to_abi_and_bytecode.py

```diff
@@ -2,7 +2,7 @@
 import json
 import os
 
-for path in glob.glob('./complete_compiled_contracts/*.json'):
+for path in glob.glob('/Users/Nick/WebstormProjects/BancorContracts/solidity/build/contracts/*.json'):
```

Comment:
@enjeyw check this isn't an accidental commit

Reply:
oops yep